joomla/joomla-cms
# Fix for: Repeatable field is no longer rendered with Chosen layout

Pull Request for Issue #16110

### Summary of Changes
The patch rerender Chosen.js if it was there before


### Testing Instructions
please see #16110 and #14556 for detail

Example xml which can be used for test:
```xml
<field name="repeatable" type="repeatable" icon="list" label="Repeatable" select="select">
	<fieldset hidden="true" name="repeatable_modal" repeat="true">

			<field type="list" name="list1" label="List1">
			<option>- Select -</option>
			<option value="val1">val 1</option>
			<option value="val2">val 2</option>
			<option value="val3">val 3</option>
		</field>

		<field type="list" name="list2" label="List2" multiple="true">
			<option>- Select -</option>
			<option value="val4">val 4</option>
			<option value="val5">val 5</option>
			<option value="val6">val 6</option>
			<option value="val7">val 7</option>
			<option value="val8">val 8</option>
		</field>
	</fieldset>
</field>

```


### Expected result
Chosen.js rendered


### Actual result
Only default `<selec>` list 


### Documentation Changes Required
none
